app/assets/stylesheets/layouts/monochrome.scss

```diff
@@ -1,23 +1,111 @@
-body { background-color: #E0E0E0; }
+@import 'variables';
+
+/* - header */
+//$font_color_header_link:  // header links
+//$font_color_header_link_hover:  // header links (hover)
+
+//$bg_color_header_top: transparent; // top header strip for logo
+//$bg_color_header_middle: #05050F; // middle header strip with user items
+//$bg_color_header_bottom: #484357; // bottom header strip with navigation
```

Comment:
leftover comments?

Reply:
I mean I would say that but looking at the file it seems like I didn't pull out the colors there?

Reply:
oh, right – are you going to do that?

Reply:
I probably should! Idk why I didn't!